modules/local/trgt/main.nf

```diff
@@ -9,7 +9,7 @@ process TRGT {
     tuple val(meta), path(bam), path(bai), val(sex)
     tuple val(meta2), path(fasta)
     tuple val(meta3), path(fai)
```

Comment:
Any idea if these be one channel or two? Meaning as you have it or like this:
`tuple val(meta2), path(fasta), path(far)`